XHR.js

```diff
@@ -0,0 +1,30 @@
+var CarLot = (function (){
+	var inventory = [];
+
+	return {
+		getInventory: function () {
+			return inventory;
+		},
+
+		setInventory: function () {
```

Comment:
Digging the naming here, this _does_ set the inventory variable, even though it's via an XHR request.